scipy/stats/contingency.py

```diff
@@ -11,13 +11,12 @@
 
    chi2_contingency
    relative_risk
+   odds_ratio
    crosstab
    association
 
    expected_freq
    margins
-   association
```

Comment:
Was removing this from the list intentional?

Reply:
Yes, because it is already listed four lines above.

Reply:
Oops, OK.